js/src/components/paid-ads/asset-group/faqs-section.js

```diff
@@ -71,11 +70,7 @@ const faqItems = [
  * @fires gla_documentation_link_click with `{ context: 'assets-faq', linkId: 'assets-faq-about-ad-formats-available-in-different-campaign-types', href: 'https://support.google.com/google-ads/answer/1722124' }`.
  */
 const FaqsSection = () => {
-	return (
-		<Section>
-			<FaqsPanel context="campaign-management" faqItems={ faqItems } />
-		</Section>
-	);
+	return <FaqsPanel context="campaign-management" faqItems={ faqItems } />;
```

Comment:
Now that you're removing the `Section` component from this and the `paid-ads/faqs-section.js` I think it would make more sense to rename them as well. Perhaps using an even a more specific name like `AssetGroupFaqsPanel` and `PaidAdsFaqsPanel`, respectively so their not confused with the primitive `FaqsPanel` component they consume.

Reply:
Handled in [0a1f579](https://github.com/woocommerce/google-listings-and-ads/pull/2531/commits/0a1f57920157f274c568fbef952e8509ac8e2f19)

Reply:
I was thinking of that! 👍